app/(main)/login/page.test.tsx

```diff
@@ -17,7 +17,13 @@ let continueButton: HTMLElement,
   emailInput: HTMLInputElement,
   passwordInput: HTMLInputElement;
 
-const mockUseAuthContext = {
+interface MockUseAuthContext {
+  getUser: jest.Mock;
+  isSignedIn: boolean | null;
+  login: jest.Mock;
+}
+
+let mockUseAuthContext: MockUseAuthContext = {
```

Comment:
Nit: The original `const` was fine since I don't think we plan on reassigning a new object to `mockUseAuthContext`.

Reply:
@CorinaMurg good point! Changed back to `const`.